grails-app/views/order/addComment.gsp

```diff
@@ -78,5 +78,6 @@
 			</div>
 		</div>
 	</div>
+	<g:render template="/dashboard/activeSection" model="[section: orderInstance.orderType.name == 'Putaway Order' ? 'inbound' : orderInstance.orderType.name == 'Outbound' ? 'outbound' : 'purchasing']"/>
```

Comment:
Rather use:
```
orderInstance?.orderType?.code == Constants.PUTAWAY_ORDER
```
or even:
```
orderInstance?.orderType?.isPutawayOrder()
```
By the way I'm also concered about comparing order to `Outbound` - we don't have `orderType` 'Outbound'.